.github/workflows/push-light.yml

```
@@ -13,11 +13,14 @@ concurrency:
 env:
   NODE_VERSION: "24"
   PNPM_VERSION: "10.32.1"
+  NODE_MODULES_CACHE_KEY: ${{ hashFiles('pnpm-lock.yaml', 'pnpm-workspace.yaml', 'packages/db/prisma/schema.prisma', 'packages/db/prisma/migrations/**', 'packages/db/prisma.config.ts') }}
```

actionlint (1.7.11)

[error] calling function "hashFiles" is not allowed here. "hashFiles" is only available in "jobs.<job_id>.steps.continue-on-error", "jobs.<job_id>.steps.env", "jobs.<job_id>.steps.if", "jobs.<job_id>.steps.name", "jobs.<job_id>.steps.run", "jobs.<job_id>.steps.timeout-minutes", "jobs.<job_id>.steps.with", "jobs.<job_id>.steps.working-directory". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)